complete/src/main/java/hello/Application.java

```diff
@@ -13,7 +13,10 @@
 public class Application extends AsyncConfigurerSupport {
 
     public static void main(String[] args) {
-        SpringApplication.run(Application.class, args);
+        SpringApplication.run(Application.class, args).close();
+
+        // Close application
+        System.exit(0);
     }
```

Comment:
I vote for the `@Bean` then. Or just not configuring at all (since the default works just fine). We could mention in the README that a `@Bean` is a logical next step if you want to customize it.